test/packages/parallel/gcp/data_stream/billing/agent/stream/stream.yml.hbs

```diff
@@ -0,0 +1,12 @@
+metricsets: ["billing"]
+period: {{period}}
+project_id: {{project_id}}
+{{#if credentials_file}}
+credentials_file_path: {{credentials_file}}
+{{/if}}
+{{#if credentials_json}}
+credentials_json: {{credentials_json}}
```

Comment:
Should this be single quoted?

Reply:
we are adding test case for `compute` data stream, so `billing` has not been updated because is not tested. (I wouldn't work anyway as it requires other changes too). I'll remove it.